[2024-07-08T09:00:52.024+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T09:00:52.112+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Final.starting manual__2024-07-08T09:00:41.388450+00:00 [queued]>
[2024-07-08T09:00:52.128+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Final.starting manual__2024-07-08T09:00:41.388450+00:00 [queued]>
[2024-07-08T09:00:52.131+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-08T09:00:52.196+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): starting> on 2024-07-08 09:00:41.388450+00:00
[2024-07-08T09:00:52.208+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=106) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-08T09:00:52.216+0000] {standard_task_runner.py:63} INFO - Started process 109 to run task
[2024-07-08T09:00:52.215+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Final', 'starting', 'manual__2024-07-08T09:00:41.388450+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/dag_test.py', '--cfg-path', '/tmp/tmpf0bcfjiy']
[2024-07-08T09:00:52.218+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask starting
[2024-07-08T09:00:52.372+0000] {task_command.py:426} INFO - Running <TaskInstance: Final.starting manual__2024-07-08T09:00:41.388450+00:00 [running]> on host e6889604c7a9
[2024-07-08T09:00:52.670+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ha20040204@gmail.com' AIRFLOW_CTX_DAG_OWNER='HA' AIRFLOW_CTX_DAG_ID='Final' AIRFLOW_CTX_TASK_ID='starting' AIRFLOW_CTX_EXECUTION_DATE='2024-07-08T09:00:41.388450+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-08T09:00:41.388450+00:00'
[2024-07-08T09:00:52.689+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T09:00:52.694+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-08T09:00:52.696+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Chuẩn bị cào dữ liệu"']
[2024-07-08T09:00:52.722+0000] {subprocess.py:86} INFO - Output:
[2024-07-08T09:00:52.732+0000] {subprocess.py:93} INFO - Chuẩn bị cào dữ liệu
[2024-07-08T09:00:52.735+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-08T09:00:52.737+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T09:00:52.808+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Final, task_id=starting, run_id=manual__2024-07-08T09:00:41.388450+00:00, execution_date=20240708T090041, start_date=20240708T090052, end_date=20240708T090052
[2024-07-08T09:00:52.922+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T09:00:53.025+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-08T09:00:53.032+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
